src/test/java/com/razorpay/CustomerClientTest.java

```diff
@@ -295,4 +296,168 @@ public void testDeleteToken() throws IOException, RazorpayException {
             assertTrue(false);
         }
     }
+
+    /**
+     * Add Bank Account
+     */
+    @Test
+    public void testaddBankAccount() throws RazorpayException{
+
+        JSONObject request = new JSONObject("{\n" +
+                "    \"ifsc_code\" : \"UTIB0000194\",\n" +
+                "    \"account_number\"         :\"916010082985661\",\n" +
+                "    \"beneficiary_name\"      : \"Pratheek\",\n" +
+                "    \"beneficiary_address1\"  : \"address 1\",\n" +
+                "    \"beneficiary_address2\"  : \"address 2\",\n" +
+                "    \"beneficiary_address3\"  : \"address 3\",\n" +
+                "    \"beneficiary_address4\"  : \"address 4\",\n" +
+                "    \"beneficiary_email\"     : \"random@email.com\",\n" +
+                "    \"beneficiary_mobile\"    : \"8762489310\",\n" +
+                "    \"beneficiary_city\"      :\"Bangalore\",\n" +
+                "    \"beneficiary_state\"     : \"KA\",\n" +
+                "    \"beneficiary_country\"   : \"IN\"\n" +
+                "}");
```

Comment:
Is there a better way to initialize the request? like instead of passing a String in the constructor, is it possible to do something like,
JSONObject request = new JSONObject();
request.put("ifsc_code", "UTIB0000194");
or may be serializing a POJO to JSONObject (is there any POJO which you can use to build the object and then converting it to JSONObject?)

because passing a string to JSONObject looks untidy

Reply:
added POJO to JSONObject

Reply:
+1